content/2017/02/04/sample.rst

```diff
@@ -0,0 +1,203 @@
+public: no
```

Comment:
Do we intend to leave this page around, just not public? If so, do we want to link to this from the Styleguide?